### ISSUE TITLE
Gitlab: Private repository can't build Docker Image

### ISSUE THREAD

**Issue by macvag** [gitpod-io/gitpod#983]
**Describe the bug**

When the repository is private isn't build the custom docker image,

**To Reproduce**

Create a private repository on Gitlab
Add .gitpod.yml 
```yaml
image:
  file: .gitpod.dockerfile
```
Add .gitpod.dockerfile
```docker
FROM gitpod/workspace-full
USER gitpod
RUN echo n | npm i @angular/cli -g
USER root
```
When the Workspace has been initialized the ng command isn't available.
Change the repository to public.
Initialize a new Workspace and you can see the logs from the building of the image and when the Workspace is opened the ng command is available.

**Expected behavior**

Build Docker Image in both occasions.

**Comment by carl-parrish** [comment]
Do we have an update on when this will be deployed?

**Comment by jankeromnes** [comment]
@carl-parrish The fix got deployed, but we simply forgot to remove the label.

**Comment by whitetigle** [comment]
Hi. it does not seem to work for me. 
I started a new private projet from gitlab and the dockerfile is never called. 
I tried in another private projet and I get exactly the same problem.
So has the fix been deployed yet? Or am I missing something?

Thanks for your help!

**Comment by whitetigle** [comment]
I tried another repository. 
- Again fresh. 
- Added non working docker configuration files. 
- It did not load. (ok since the docker file is not correct)
- I updated the docker file to be correct and pushed changes to the repo
- It did not load. It was still complaining about the same issue
- I deleted the project from the workspace
- I loaded it again using `gitpod.io#/https//gitlab....`
- now it works. 

So I guess it's more a cache problem
